src/index.ts

```diff
@@ -90,6 +94,60 @@ async function setDebugSession(
   app.commands.notifyCommandChanged();
 }
 
+function updateToolbar(
+  widget: NotebookPanel | ConsolePanel | FileEditor,
+  debug: IDebugger,
+  commands: CommandRegistry,
+  handler: any
+): void {
+  const isConsolePanel = widget instanceof ConsolePanel;
+  const checkState = () =>
+    debug.session.client.name === debug.session.currentStateClient;
+
+  widget.node.setAttribute('data-debugger-on', checkState().toString());
+
+  const getToolbar = (): Toolbar => {
+    if (isConsolePanel) {
+      return (
+        ((widget as ConsolePanel).widgets.find(
+          widget => widget instanceof Toolbar
+        ) as Toolbar) || new Toolbar()
+      );
+    } else {
+      return (widget as NotebookPanel)?.toolbar;
```

Comment:
If the `?` is necessary here, should we also have it above? (for example for `widget as ConsolePanel`)